### PR TITLE
feat: Apply malware disabled rules

### PR DESCRIPTION
**Pull request by mhuth**
* Download disabled malware rules list
* Skip any matches on the disabled rules
* Make disabled rules case-insensitive

### All Pull Requests:

Check all that apply:

* [X] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [X] Is this PR an enhancement?

### Complete Description of Additions/Changes:
This PR downloads the list of disabled malware rules (if any) from the malware backend and skips any matches for these disabled rules, so those matches don't show up in the results archive that's sent back.

### CODE REVIEW

**Comment by mhuth** [comment]
@xiangce Any chance you can approve / merge this.  I'm sure its going to be ok even if Dan doesn't review it.

**Review by xiangce — APPROVED** [review]
Yup, I was waiting for Dan's Approval. :)

**Pull request merged**
